android/settings.gradle

```diff
@@ -0,0 +1,2 @@
+include ':app'
+rootProject.name = "TF Segement + Stylize"
```

Comment:
As @margaretmz suggested, "Segement" -> "Segment", we can comeup with a proper name later.

Reply:
Fixed :100: